test/functional/simple_smart_answers_controller_test.rb

```diff
@@ -41,6 +41,30 @@ def simple_smart_answer_content_item_with_start_button_text
       end
     end
 
+    context "tasklist header A/B testing" do
+      setup do
+        content_store_has_random_item(base_path: '/learn-to-drive-dangerously', schema: 'simple_smart_answer')
+
+        @controller.stubs(:tasklist_header_ab_test_applies?).returns(true)
+      end
+
+      should "not show the tasklist header by default" do
+        with_variant TaskListHeader: "A" do
+          get :show, slug: "learn-to-drive-dangerously"
+
+          assert_template partial: "_tasklist_header", count: 0
+        end
+      end
+
+      should "show the tasklist header for the 'B' version" do
+        with_variant TaskListHeader: "B" do
```

Comment:
Could we have a test to make sure that the header doesn't get rendered (in the B variant) if `tasklist_header_ab_test_applies?` is false?

Reply:
Thanks @sihugh , I have added this in now.